utils/_context/_scenarios/__init__.py

```diff
@@ -249,6 +249,19 @@ class _Scenarios:
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    appsec_auto_events_blocking = EndToEndScenario(
```

Comment:
Why using a new scenario? this one seems exactly the same as `appsec_runtime_activation`

Reply:
edit: I'm reading more carefully the PR description, to see if it answer my question

Reply:
appsec activation has appsec disabled at the start to be able to activate it at runtime. For this test we need a scenario that has RC but appsec already activated. Maybe there is already one that fits and we don't need to add a new one

Reply:
If we want - I can rename the scenario to be explicit and not tied to any feature. For some reason `APPSEC_REQUEST_BLOCKING` didn't work for all languages (maybe something with its setup)

I'm open to any name here